purchase_work_acceptance_invoice_plan/wizard/select_work_acceptance_invoice_plan_wizard.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import ValidationError
+
+
+class SelectWorkAcceptanceInvoicePlanWizard(models.TransientModel):
+    _name = "select.work.acceptance.invoice.plan.wizard"
+    _description = "Select Work Acceptance Invoice Plan Wizard"
+
+    require_wa = fields.Boolean(default=lambda self: self._get_require_wa())
+    wa_in_id = fields.Many2one(
+        comodel_name="purchase.invoice.plan",
+        string="Invoice Plan",
+        domain=lambda self: [
```

Comment:
I think it will be better if we list only the invoice plan that has not bee used.
```suggestion
        domain="[('id', 'in', 'active_invoice_plan_ids')]",
```

Reply:
When create the compute field, i.e.,

active_invoice_plan_ids = fields.Many2many(comodel_name="purchase.invoice.plan", compute="....")

then, compute to get the invoice_plan_ids that is still ok to us.